templates/laravel/files/.platform.app.yaml

```diff
@@ -24,22 +24,36 @@ hooks:
 # to the application in the PLATFORM_RELATIONSHIPS variable. The right-hand
 # side is in the form `<service name>:<endpoint name>`.
 relationships:
-    database: "mysqldb:mysql"
-    rediscache: "redis:redis"
-    redissession: "redis:redis"
+    database: "db:mysql"
+    rediscache: "cache:redis"
+    redissession: "cache:redis"
 
 # The size of the persistent disk of the application (in MB).
 disk: 2048
 
 # The mounts that will be performed when the package is deployed.
 mounts:
-  "storage/app/public": "shared:files/public"
-  "storage/framework/views": "shared:files/views"
-  "storage/framework/sessions": "shared:files/sessions"
-  "storage/framework/cache": "shared:files/cache"
-  "storage/logs": "shared:files/logs"
-  "bootstrap/cache": "shared:files/bootstrap/cache"
-  "/.config": "shared:files/config"
+  "storage/app/public":
+      source: local
+      source_path: "public"
+  "storage/framework/views":
+      source: local
+      source_path: "views"
+  "storage/framework/sessions":
+      source: local
+      source_path: "sessions"
+  "storage/framework/cache":
+      source: local
+      source_path: "cache"
+  "storage/logs":
+      source: local
+      source_path: "logs"
+  "bootstrap/cache":
+      source: local
+      source_path: "cache"
```

Comment:
```suggestion
      source_path: "bootstrap/cache"
```

Reply:
Is this necessary?  Again, the code doesn't matter but it seems like just unnecessary complexity to me.

Reply:
Indeed it does, I think I was just trying to retain what was there and not thinking how that would add unneeded complexity to source.